pom.xml

```diff
@@ -3,11 +3,13 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.smartbear.readyapi</groupId>
-    <artifactId>ready-api-testserver-client</artifactId>
-    <version>1.2.2-SNAPSHOT</version>
+    <artifactId>readyapi4j-base</artifactId>
```

Comment:
This should be simply readyapi4j, and the module called readyapi4j should be readyapi4j-facade (Prakash agrees).